### PR TITLE
Update the Yocto training STM32: for Dunfell and with the DK2, add some details in slides

### PR DESCRIPTION
**Pull request by tprrt**

### CODE REVIEW

File: slides/yocto-advanced/yocto-advanced.tex [hunk 1]
```diff
@@ -205,7 +205,7 @@ \subsection{Packages variants}
   \frametitle{Version selection}
   \begin{itemize}
     \item By default, Bitbake will try to build the provider with the
-      highest version number, unless the recipe defines
+      highest version number, from layer with the highest priority, unless the recipe defines
```
Comment (alexandrebelloni) [inline slides/yocto-advanced/yocto-advanced.tex:208]: "from the highest priority layer" also, please wrap that line properly

**Review by alexandrebelloni — CHANGES_REQUESTED** [review]

File: slides/yocto-advanced/yocto-advanced.tex [hunk 2]
```diff
@@ -308,8 +308,13 @@ \subsection{The power of BitBake}
...
         \begin{minted}[fontsize=\footnotesize]{console}
-$ ./scripts/sstate-cache-management.sh --remove-duplicated -d \
-  --cache-dir=<SSTATE_DIR>
+$ ./scripts/sstate-cache-management.sh -y -d \
+  --cache-dir=$SSTATE_DIR
+        \end{minted}
+      \end{block}
+      \begin{block}{}
+        \begin{minted}[fontsize=\footnotesize]{console}
+$ find -atime +30 -delete $SSTATE_DIR
```
Comment (alexandrebelloni) [inline slides/yocto-advanced/yocto-advanced.tex:317]: This doesn't necessarily work on filesystem mounted with noatime which is very common on build machines so at least add a description of what this does

**Review by alexandrebelloni — CHANGES_REQUESTED** [review]

**Review by alexandrebelloni — APPROVED** [review]

**Review by alexandrebelloni — APPROVED** [review]

**Review by alexandrebelloni — APPROVED** [review]

File: slides/yocto-image/yocto-image.tex [hunk 3]
```diff
@@ -186,6 +186,7 @@ \subsection{Package groups}
       \code{meta*/recipes-core/packagegroups/}
     \item A package group is yet another recipe.
     \item The prefix \code{packagegroup-} is always used.
+    \item Be careful about the \code{PACKAGE_ARCH} value.
```
Comment (alexandrebelloni) [inline slides/yocto-image/yocto-image.tex:189]: This needs more details

**Review by alexandrebelloni — CHANGES_REQUESTED** [review]

File: slides/yocto-recipe-extra/yocto-recipe-extra.tex [hunk 4]
```diff
@@ -124,7 +124,7 @@ \subsection{Python tasks}
   \frametitle{Python task example}
   \begin{minted}{python}
 # Anonymous function
-python () {
+python __anonymous() { # or python () {
```
Comment (alexandrebelloni) [inline slides/yocto-recipe-extra/yocto-recipe-extra.tex:127]: I'll settle for simply `python __anonymous()`

**Review by alexandrebelloni — APPROVED** [review]

**Review by alexandrebelloni — APPROVED** [review]

File: slides/yocto-sdk/yocto-sdk.tex [hunk 5]
```diff
@@ -273,7 +273,7 @@ \subsection{Devtool}
         \item \code{fetchuri} can be a local path or a remote {\em
           uri}.
       \end{itemize}
-    \item To modify an existing recipe: \code{devtool modify <recipe>}
+    \item To modify the source of an existing recipe: \code{devtool modify <recipe>}
```
Comment (alexandrebelloni) [inline slides/yocto-sdk/yocto-sdk.tex]: devtool modify allows to both modify the recipe itself and the sources of the targeted application so I'm not sure about that change

File: labs/yocto-advanced-configuration-stm32/yocto-advanced-configuration-stm32.tex [hunk 6]
```diff
@@ -18,9 +18,9 @@ \section{Set up the Ethernet communication and NFS on the board}
 
 First we need to set the kernel boot arguments U-Boot will pass to the
 Linux kernel at boot time. For that, edit the
-\code{mmc0_stm32mp157a-dk1_extlinux/extlinux.conf} file in the bootfs partition of the SD
-card and change the \code{APPEND} line for the
-\code{stm32mp157a-dk1-sdcard} configuration:
+\code{mmc0_extlinux/extlinux.conf}, or \code{mmc0_extlinux/stm32mp157c-dk2_extlinux.conf}
+file, in the bootfs partition of the SD card and change the \code{APPEND} line
+for the \code{stm32mp157a-dk1-sdcard} configuration:
```
Comment (alexandrebelloni) [inline labs/yocto-advanced-configuration-stm32/yocto-advanced-configuration-stm32.tex:23]: I would think `\code{stm32mp157a-dk1-sdcard}` also has to be changed

**Review by alexandrebelloni — APPROVED** [review]

File: slides/yocto-image/yocto-image.tex (hunk 3 above)
Comment (tprrt) [reply slides/yocto-image/yocto-image.tex]: Added some details about PACKAGE_ARCH

**Comment by alexandrebelloni** [comment]
Merged

**Pull request closed without merging**